tree_utils.go

```diff
@@ -0,0 +1,31 @@
+package i3
+
+import "strings"
+
+func (n *Node) FindFocusedLeaf() *Node {
+	return n.FindFocused(func(n *Node) bool { return n.Focused })
+}
+
+func (child *Node) FindParent() *Node {
```

Comment:
I think this one is okay. Can you please add a comment?

Reply:
Fixed

Reply:
typo: curren → current

Also, can you mention “by requesting and traversing the tree” so that people get an idea of the performance characteristics?

Reply:
Done